pages/templatetags/pages_extras.py

```diff
@@ -9,63 +9,19 @@
 
 
 @register.inclusion_tag('pages/breadcrumbs.html')
-def breadcrumbs(page: Page, separator='', base_url=''):
-    index = page.get_index()
-
-    crumbs_list = (
-        (index.display_menu_title, index.url) if index else ('Main', '/'),
-        *page.get_ancestors_fields('display_menu_title', 'url', include_self=False),
-        (page.display_menu_title, '')
-    )
-
-    return {
-        'crumbs_list': crumbs_list,
-        'separator': separator,
-        'base_url': base_url,
-    }
-
-
-@register.inclusion_tag('pages/breadcrumbs_with_siblings.html')
-def breadcrumbs_with_siblings(
-    page: Page, separator='', base_url='', include_self=False
-):
-    def get_ancestors_crumbs() -> list:
-        ancestors_query = (
-            page
-            .get_ancestors(include_self)
-            .select_related(page.related_model_name)
-            .active()
-        )
-
-        if not ancestors_query.exists():
-            return []
-
-        catalog, *ancestors = ancestors_query
-
-        return [
-            (catalog.display_menu_title, catalog.url, []),
-            *[
-                (
-                    crumb.display_menu_title,
-                    crumb.url,
-                    list(logic.Page(page).siblings)
-                ) for crumb in ancestors
-            ],
-        ]
-
-    index = page.get_index()
-
-    crumbs_list = [
-        (index.display_menu_title, index.url, []) if index else ('Main', '/', []),
-        *get_ancestors_crumbs(),
-        (page.display_menu_title, '', logic.Page(page).siblings)
-    ]
-
+def breadcrumbs(page: Page, separator='', show_siblings=False):
```

Comment:
I believe it would be much better to get rid this cryptic `show_siblings` and create two separate functions `breadcrumbs` and `breadcrumbs_with_siblings`